### PR TITLE
[docs] Update MapView documentation

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Instructions in `react-native-naps` documentation currently is outdated on various levels. Working with development builds is also not highlighted and the text is confusing when following instructions to generate API key for Google Maps on Android.

Closes ENG-6618, ENG-6596, ENG-6216

# How

This PR removes the outdated references to standalone apps and replaces that with development builds. It also backport these changes to SDK 47 and 46.

Also, removes duplicated step for web configuration and removes [How to retrieve your debug keystore fingerprint (Android only)](https://docs.expo.dev/versions/latest/sdk/map-view/#how-to-retrieve-your-debug-keystore-fingerprint).

# Test Plan

Changes have been tested by running docs locally and visiting: http://localhost:3002/versions/latest/sdk/map-view

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/map-view.mdx [hunk 1]
```diff
@@ -52,98 +56,104 @@ const styles = StyleSheet.create({
...
+- In your code, import `{ PROVIDER_GOOGLE }` from `react-native-maps` and add the property `provider=PROVIDER_GOOGLE` to your `<MapView>`. This property works on both Android and iOS.
+- Rebuild the app binary. An easy way to test if the configuration was successful is to do a [simulator build](/development/create-development-builds/#create-and-install-a-build-for-emulatorsimulator).
+
+</Step>
 
-## Configuring for web
+## Configuration for web
 
-> Web is experimental! We do not recommend using this library on web yet.
+> Web is experimental! We do not recommend using this library on the web yet.
```
Comment (Simek) [inline docs/pages/versions/unversioned/sdk/map-view.mdx:154]:
```suggestion
> **warning** Web support is experimental! We do not recommend using this library on the web yet.
```

Note formatting.

File: docs/pages/versions/unversioned/sdk/map-view.mdx [hunk 2]
```diff
@@ -52,98 +56,104 @@ const styles = StyleSheet.create({
...
 
-To use this in web, add the following script to your **web/index.html**. This script may already be present, if this is the case, just replace the `API_KEY` with your Google Maps API key which you can obtain here: [Google Maps: Get API key](https://developers.google.com/maps/documentation/javascript/get-api-key)
+To use this on the web, add the following script to your **web/index.html**. This script may already be present. If this is the case, replace the `API_KEY` with your Google Maps API key, which you can obtain here: [Google Maps: Get API key](https://developers.google.com/maps/documentation/javascript/get-api-key).
 
 ```html
```
Comment (Simek) [inline docs/pages/versions/unversioned/sdk/map-view.mdx:158]:
```suggestion
```html web/index.html
```

**Review by Simek — APPROVED** [review]
This is a nice improvement, thank you! 👍 

Left a few formatting comments, and if you want, we can simplify the basic usage a bit:
* https://snack.expo.dev/@simek/mapview

**Comment by cjones26** [comment]
@amandeepmittal why was this section removed: https://github.com/expo/expo/pull/20136/files#diff-65685780b3b5eea89044cb92ce7f0479e870d14584e39e11b4184d62523298e9L101?

I had to dig through old issues and docs to figure out exactly how where the signing key was coming from, as I'm not that familiar with Android development.

I think it's important for folks to understand that the same debug key is utilized for _all_ development builds which use prebuild so they can account for it accordingly.

There also doesn't seem to be a clear way to replace this file with a custom debug keystore.  Is there any documentation around that, or do we need to write a custom config plugin / create our own custom template?  I can't seem to get `credentials.json` to work with local development (ex. `npx expo run:android`).

**Comment by cjones26** [comment]
For anyone else running into this, here was our solution:

```
import { ConfigPlugin, withDangerousMod } from 'expo/config-plugins';
import fs from 'fs';
import path from 'path';

export const withAndroidKeystore: ConfigPlugin<{
  sourceKeystorePath: string;
}> = (config, { sourceKeystorePath }) => {
  if (!sourceKeystorePath) {
    throw new Error(
      'No source keystore path was provided to the config plugins.',
    );
  }

  return withDangerousMod(config, [
    'android',
    (config) => {
      const fullKeystorePath = path.join(
        config.modRequest.projectRoot,
        sourceKeystorePath,
      );

      fs.copyFileSync(
        fullKeystorePath,
        path.join(
          path.join(config.modRequest.platformProjectRoot, 'app'),
          path.basename(sourceKeystorePath),
        ),
      );

      return config;
    },
  ]);
};
```

**Pull request merged**